scripts/py_matter_yamltests/matter/yamltests/websocket_runner.py

```diff
@@ -105,7 +106,13 @@ async def _start_server(self, command, url):
         if command:
             start_time = time.time()
 
-            command = ['stdbuf', '-o0', '-e0'] + command  # disable buffering
+            # Disable buffering for live test output, if 'stdbuf' is available.
+            # This improves real-time logs in WebSocketRunner.
+            if shutil.which("stdbuf"):
+                command = ['stdbuf', '-o0', '-e0'] + command
+            else:
+                logging.debug("'stdbuf' not found - output may be buffered.")
```

Comment:
But we're doing pattern-matching on that output to decide when to return from this function, no?  If the output is buffered, we might not see the _WEBSOCKET_SERVER_MESSAGE because it's in the buffer and will just time things out even though the server is started.  This does not look right at all.

But why are we using stdbuf here to start with?  Why not just pass the appropriate `bufsize` (either 0 or 1 depending on how that interacts with the `readline()` call below) instead?